examples/with-flow/types/next.js.flow

```diff
@@ -0,0 +1,41 @@
+declare module "next" {
+  declare type NextApp = {
+    prepare(): Promise<void>;
```

Comment:
@nkzawa Thank you for your notice!
I wrote those types along with `README.md` and read codes a bit, but I lost them. (It looks not documented yet.) Anyway I add them.